src/JwksHandler.ts

```diff
@@ -11,16 +11,12 @@ import * as AWS from "@aws-sdk/client-kms";
 
 const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : "DEBUG";
 const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.JWKS_LOGGER_SVC_NAME;
-const logger = new Logger({
+export const logger = new Logger({
 	logLevel: POWERTOOLS_LOG_LEVEL,
 	serviceName: POWERTOOLS_SERVICE_NAME,
 });
 
-const SIGNING_KEY_IDS = process.env.SIGNING_KEY_IDS;
-const ENCRYPTION_KEY_IDS = process.env.ENCRYPTION_KEY_IDS;
-const JWKS_BUCKET_NAME = process.env.JWKS_BUCKET_NAME;
-
-const s3Client = new S3Client({
+export const s3Client = new S3Client({
```

Comment:
nit: We could access all these using . notation as private methods instead of exporting

Reply:
Done, but I'd appreciate a fairly thorough review on the most recent commit just to be sure please 🙏